articles/libraries/auth0js/index.md

```diff
@@ -1,28 +1,28 @@
 ---
 section: libraries
 toc: true
-description: How to install, initialize and use auth0.js
+description: How to install, initialize and use auth0.js v8
 url: /libraries/auth0js
 ---
 
-# Auth0.js Reference
+# Auth0.js v8 Reference
 
 Auth0.js is a client-side library for [Auth0](http://auth0.com), for use in your web apps. It allows you to trigger the authentication process and parse the [JSON Web Token](http://openid.net/specs/draft-jones-json-web-token-07.html) (JWT) with just the Auth0 `clientID`. Once you have the JWT, you can use it to authenticate requests to your HTTP API and validate the JWT in your server-side logic with the `clientSecret`.
```

Comment:
Yes, let's focus on what auth0.js does which is making easier to do Authentication and Authorization in your web apps with Auth0. 
Also let's omit the JWT part which should not matter to the user unless he is working with OIDC.
A spin to the `Once you have the JWT, you can use it to authenticate requests to your HTTP API and validate the JWT in your server-side logic with the clientSecret.` is that after Auth with auth0.js you can get a token to use with your own API (we can link to the api auth docs if needed)